src/shared/components/XYPlot.tsx

```diff
@@ -220,8 +228,23 @@ const XYPlot: FC<Props> = ({
     ],
   }
 
-  if (!isValidView) {
-    return <EmptyGraphMessage message={INVALID_DATA_COPY} />
+  if (isFlagEnabled('annotations')) {
+    const doubleClickHandler = plotInteraction => {
```

Comment:
is there a type for plotInteraction?

Reply:
https://github.com/influxdata/giraffe/blob/master/giraffe/src/types/index.ts#L88-L101